yarpl/include/yarpl/flowable/Subscribers.h

```diff
@@ -62,6 +62,23 @@ class Subscribers {
         std::move(next), std::move(error), std::move(complete), batch);
   }
 
+  template <
+      typename T,
+      typename Sub,
+      typename Next,
+      typename Error,
+      typename Complete,
+      typename = typename std::enable_if<
+          std::is_callable<Sub(Reference<Subscription>), void>::value &&
```

Comment:
I really gotta get on that `s|enable_if|static_assert|g` killing spree I've been planning on doing...